addons/mrp/views/mrp_production_views.xml

```diff
@@ -279,17 +278,18 @@
                                     <field name="state" invisible="1" force_save="1"/>
                                     <field name="should_consume_qty" invisible="1"/>
                                     <field name="product_uom_qty" widget="mrp_should_consume" string="To Consume" attrs="{'readonly': ['&amp;', ('parent.state', '!=', 'draft'), '|', '&amp;', ('parent.state', 'not in', ('confirmed', 'progress', 'to_close')), ('parent.is_planned', '!=', True), ('parent.is_locked', '=', True)]}" width="1"/>
+                                    <field name="product_uom" attrs="{'readonly': [('state', '!=', 'draft')]}" options="{'no_open': True, 'no_create': True}" groups="uom.group_uom"/>
                                     <field name="product_type" invisible="1"/>
                                     <field name="product_qty" invisible="1" readonly="1"/>
                                     <field name="reserved_availability" invisible="1"/>
                                     <field name="forecast_expected_date" invisible="1"/>
-                                    <field name="forecast_availability" string="Reserved" attrs="{'column_invisible': [('parent.state', 'in', ['done'])]}" widget="forecast_widget"/>
+                                    <field name="forecast_availability" string="Availability" attrs="{'column_invisible': [('parent.state', 'in', ('draft', 'done'))]}" widget="forecast_widget"/>
```

Comment:
I thought of doing this:
```xml
<field name="forecast_availability" string="Availability" attrs="{'column_invisible': [('parent.state', 'in', ('draft', 'done'))]}" widget="forecast_widget"/>
<field name="forecast_availability" string="Reserved" attrs="{'column_invisible': [('parent.state', 'not in', ('draft'))]}" widget="forecast_widget"/>
```
But it's doing a weird behaviour:

![image](https://user-images.githubusercontent.com/32851253/128353652-460bdaa0-6101-4421-8a33-2bdc95e5abd0.png)
And:
![image](https://user-images.githubusercontent.com/32851253/128353710-33a69528-9fe9-4c94-ae74-b063b9442aa4.png)

Reply:
It will be a good idea to change the name of the variable?
Because `forecast_availability` for something that is reserved, when there is another variable called `forecast_available` it's not super nice?